qtpy/__init__.py

```diff
@@ -26,42 +63,42 @@
     >>> from qtpy import QtGui, QtWidgets, QtCore
     >>> print(QtWidgets.QWidget)
 
-
 PySide2
-======
+=======
 
-Set the QT_API environment variable to 'pyside2' before importing other
+Set the QT_API environment variable to 'PySide2' before importing other
 packages::
 
     >>> import os
-    >>> os.environ['QT_API'] = 'pyside2'
+    >>> os.environ['QT_API'] = 'PySide2'
```

Comment:
Sorry but I don't have much time right now to do a thorough review of this. For now, please revert this change you made here about using capitalization to set `QT_API`.

There's a well established convention among Python projects that deal with different Qt bindings to use `QT_API` as we are using it here, so please revert that.